Microsoft.Toolkit.Uwp/Helpers/RomeHelper/RomeHelper.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Windows.System.RemoteSystems;
+
+namespace Microsoft.Toolkit.Uwp.Helpers
+{
+    /// <summary>
+    /// Helper to List Remote Devices that are accessible
+    /// </summary>
+    public class RomeHelper
```

Comment:
Should this be called RemoteDeviceHelper to match the name of the control?

Reply:
Changed it to `RemoteDeviceHelper`